src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,57 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int MAX_RANDOM_PROPERTIES = 50;
+
+    public Figure getRandomFigure() {
+        int randomFigureValue = new Random().nextInt(NameOfFigure.values().length);
```

Comment:
Random should be created once as private final field on class level, not each time here
```suggestion
        int randomFigureValue = random.nextInt(NameOfFigure.values().length);
```